include/cute/arch/copy_xe.hpp

```diff
@@ -113,366 +117,2211 @@ SYCL_DEVICE_BUILTIN(void __builtin_IB_subgroup_block_read_prefetch_u16_m32k16v2(
+        long baseoffset, int width_minus_one, int height_minus_one,
+        int pitch_minus_one, intel::coord_t coord));
+SYCL_DEVICE_BUILTIN(
+    intel::ushort64 __builtin_IB_subgroup_block_read_flat_u16_m32k16v2(
+        long baseoffset, int width_minus_one, int height_minus_one,
+        int pitch_minus_one, intel::coord_t coord));
+
+// 32bits specific for tf32 No transform No transpose
+SYCL_DEVICE_BUILTIN(
+    uint __builtin_IB_subgroup_block_read_flat_u32_m1k8v1(
+        long baseoffset, int width_minus_one, int height_minus_one,
+        int pitch_minus_one, intel::coord_t coord));
+SYCL_DEVICE_BUILTIN(
+    uint __builtin_IB_subgroup_block_read_flat_u32_m2k8v1(
+        long baseoffset, int width_minus_one, int height_minus_one,
+        int pitch_minus_one, intel::coord_t coord));
+SYCL_DEVICE_BUILTIN(
+    intel::uint2 __builtin_IB_subgroup_block_read_flat_u32_m4k8v1(
+        long baseoffset, int width_minus_one, int height_minus_one,
+        int pitch_minus_one, intel::coord_t coord));
+SYCL_DEVICE_BUILTIN(
+    intel::uint4 __builtin_IB_subgroup_block_read_flat_u32_m8k8v1(
+        long baseoffset, int width_minus_one, int height_minus_one,
+        int pitch_minus_one, intel::coord_t coord));
+SYCL_DEVICE_BUILTIN(
+    intel::uint8 __builtin_IB_subgroup_block_read_flat_u32_m16k8v1(
+        long baseoffset, int width_minus_one, int height_minus_one,
+        int pitch_minus_one, intel::coord_t coord));
+SYCL_DEVICE_BUILTIN(
+    intel::uint16 __builtin_IB_subgroup_block_read_flat_u32_m32k8v1(
+        long baseoffset, int width_minus_one, int height_minus_one,
+        int pitch_minus_one, intel::coord_t coord));
+
+SYCL_DEVICE_BUILTIN(
+    intel::uint2 __builtin_IB_subgroup_block_read_flat_u32_m1k8v2(
+        long baseoffset, int width_minus_one, int height_minus_one,
+        int pitch_minus_one, intel::coord_t coord));
+SYCL_DEVICE_BUILTIN(
+    intel::uint2 __builtin_IB_subgroup_block_read_flat_u32_m2k8v2(
+        long baseoffset, int width_minus_one, int height_minus_one,
+        int pitch_minus_one, intel::coord_t coord));
+SYCL_DEVICE_BUILTIN(
+    intel::uint4 __builtin_IB_subgroup_block_read_flat_u32_m4k8v2(
+        long baseoffset, int width_minus_one, int height_minus_one,
+        int pitch_minus_one, intel::coord_t coord));
+SYCL_DEVICE_BUILTIN(
+    intel::uint8 __builtin_IB_subgroup_block_read_flat_u32_m8k8v2(
+        long baseoffset, int width_minus_one, int height_minus_one,
+        int pitch_minus_one, intel::coord_t coord));
+SYCL_DEVICE_BUILTIN(
+    intel::uint16 __builtin_IB_subgroup_block_read_flat_u32_m16k8v2(
+        long baseoffset, int width_minus_one, int height_minus_one,
+        int pitch_minus_one, intel::coord_t coord));
+SYCL_DEVICE_BUILTIN(
+    intel::uint32 __builtin_IB_subgroup_block_read_flat_u32_m32k8v2(
+        long baseoffset, int width_minus_one, int height_minus_one,
+        int pitch_minus_one, intel::coord_t coord));
+
+// 32bits No transform No transpose
```

Comment:
I was wondering if it would best to split all these declarations into separate files, 
for example `xe_copy_8B.hpp`, `xe_copy_4B.hpp` and so on, and include those files here in `copy_xe.hpp`. That way this file will will only have the `struct` declarations, which will make it easier to read and understand and look back at it later